assets/front-end/css/view/fancy-text.css

```diff
@@ -30,7 +30,7 @@
     opacity: 1;
   }
 }
-@keyframes blink_cursor {
+@keyframes eaelBlink_cursor {
```

Comment:
For these type of animations we may need to update later.
Format should be "first word capital and no underscore".
In this case: `eaelBlinkCursor`